src/99-appendices/05-units.md

```diff
@@ -1,39 +1,78 @@
-| mole           | mol         | amount of substance                        |
-| candela        | cd          | luminous intensity                         |
-| radian         | rad         | angle                                      |
-| steradian      | sr          | solid angle                                |
-| hertz          | Hz          | frequency                                  |
-| newton         | N           | force, weight                              |
-| pascal         | Pa          | pressure, stress                           |
-| joule          | J           | energy, work, heat                         |
-| watt           | W           | power, radiant flux                        |
-| coulomb        | C           | electric charge or quantity of electricity |
-| volt           | V           | voltage (electrical potential), emf        |
-| farad          | F           | capacitance                                |
-| ohm            | Ω           | resistance, impedance, reactance           |
-| siemens        | S           | electrical conductance                     |
-| weber          | Wb          | magnetic flux                              |
-| tesla          | T           | magnetic flux density                      |
-| henry          | H           | inductance                                 |
-| degree Celsius | °C          | temperature relative to 273.15 K           |
-| lumen          | lm          | luminous flux                              |
-| lux            | lx          | illuminance                                |
-| becquerel      | Bq          | radioactivity (decays per unit time)       |
-| gray           | Gy          | absorbed dose (of ionizing radiation)      |
-| sievert        | Sv          | equivalent dose (of ionizing radiation)    |
-| katal          | kat         | catalytic activity                         |
+The [CMIXF-12](https://people.csail.mit.edu/jaffer/MIXF/CMIXF-12) convention
+for encoding units is REQUIRED to achieve maximum portability and limited
+variability of representation.
+If a CMIXF-12 representation of a unit is not possible, the unit MUST be declared
+as custom units and defined in an accompanying JSON file, as described in the
+[units section](../02-common-principles.md#units).
+Earlier versions of the BIDS standard listed the following Unicode symbols, and
+these are still included for backwards compatibility:
+
+1.  [`U+00B5` (µ)](https://codepoints.net/U+00B5)
+1.  [`U+00B0` (°)](https://codepoints.net/U+00B0)
+1.  [`U+2126` (Ω)](https://codepoints.net/U+2126)
+
+It is REQUIRED that units be CMIXF-12 compliant or among these three Unicode
+characters.
+Please take care not to confuse the accepted Unicode character
+[`U+00B5`](https://codepoints.net/U+00B5) (µ) with the disallowed Unicode
+character [`U+03BC`](https://codepoints.net/U+03BC) (μ):
+They look equal in appearance but are different to the computer.
```

Comment:
Since U+00B5 maps to U+03BC but not the other way around, shouldn't U+03BC be the preferred value? That way data files which are modified by software using the compatibility mapping don't need to be manually fixed after they are modified.

Reply:
thanks @nellh that sounds reasonable - I have not heard of this mapping before -> do you have a reference where I could read up on this?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize for JavaScript.

https://docs.python.org/3/library/unicodedata.html#unicodedata.normalize for Python.

Wikipedia also has a general summary under https://en.m.wikipedia.org/wiki/Unicode_equivalence

Reply:
> Since U+00B5 maps to U+03BC but not the other way around

Okay I get it now. Here is a Python example:

```Python
import unicodedata

# get utf-8 encoding of MICRO sign after NFC norm
unicodedata.normalize('NFC', 'µ').encode()                             
b'\xc2\xb5'

# same but NFKC norm --> MICRO sign is now SMALL GREEK MU
unicodedata.normalize('NFKC', 'µ').encode()                           
b'\xce\xbc'

# try same for SMALL GREEK MU: NFC norm doesn't change anything
unicodedata.normalize('NFKC', 'μ').encode()                           
b'\xce\xbc'

# also NFKC norm doesn't change anything
unicodedata.normalize('NFC', 'μ').encode()                            
b'\xce\xbc'
```